.github/workflows/run_test_case.yaml

```diff
@@ -88,3 +90,8 @@ jobs:
       with:
         name: stream.log
         path: log/stream.log
+
+  run_fvt_tests:
+    needs:
+      - run_unit_tests
+    uses: ./.github/workflows/run_fvt_tests.yaml
```

Comment:
Is it work ? Is this the new feature for Github workflow ?

Reply:
Yes, it works:

<img width="624" alt="Screenshot 2023-05-30 at 4 31 58 PM" src="https://github.com/lf-edge/ekuiper/assets/28824352/3da6ba95-8fe1-45f6-acd4-e026e4230ed8">

This feature exists for a while I guess.